content/intro-to-storybook/react/fr/test.md

```diff
@@ -0,0 +1,121 @@
+- **Les tests unitaires** avec Jest vérifient que la sortie d'un composant reste la même avec une entrée fixe. Ils sont parfaits pour tester les qualités fonctionnelles d'un composant.
+
+## "Mais est-ce que ça a l'air bien ?"
+
+Malheureusement, les méthodes de test mentionnées ci-dessus ne suffisent pas à prévenir les bugs de l'interface utilisateur. Les interfaces utilisateur sont délicates à tester car la conception est subjective et nuancée. Les tests manuels sont, eh bien, manuels. Les tests instantanés déclenchent trop de faux positifs lorsqu'ils sont utilisés pour l'interface utilisateur. Les tests unitaires au niveau pixel sont de mauvaise qualité. Une stratégie de test complète de Storybook comprend également des tests de régression visuelle.
+
+## Tests visuels pour Storybook
+
+Les tests de régression visuelle, également appelés tests visuels, sont conçus pour détecter les changements d'apparence. Ils fonctionnent en prenant des captures d'écran de chaque story et en les comparant entre elles commit par commit pour faire ressortir les changements. C'est parfait pour vérifier des éléments graphiques comme la mise en page, la couleur, la taille et le contraste.
+
+<video autoPlay muted playsInline loop style="width:480px; margin: 0 auto;">
+  <source
+    src="/intro-to-storybook/visual-regression-testing.mp4"
+    type="video/mp4"
+  />
+</video>
+
+Storybook est un outil fantastique pour les tests de régression visuelle car chaque story est essentiellement une spécification de test. Chaque fois que nous écrivons ou mettons à jour un story, nous obtenons gratuitement une spécification !
+
+Il existe un certain nombre d'outils pour les tests de régression visuelle. Nous recommandons [**Chromatic**](https://www.chromatic.com/), un service de publication gratuit créé par les mainteneurs de Storybook qui effectue des tests visuels dans un cloud parallélisé. Il nous permet également de publier Storybook en ligne, comme nous l'avons vu dans le [chapitre précédent](/react/fr/deploy/).
+
+## Voir les modifications de l'interface utilisateur
+
+Les tests de régression visuelle reposent sur la comparaison des images du nouveau code UI rendu avec les images de base. Si une modification de l'interface utilisateur est détectée, nous en sommes informés.
+
+Voyons comment cela fonctionne en modifiant l'arrière-plan du composant `Tâche`.
+
+Commencez par créer une nouvelle branche pour ce changement :
+
+```bash
+git checkout -b change-task-background
+```
+
+Modifiez `Task` comme suit:
+
+```js
+// src/components/Task.js
+<div className="title">
+  <input
+    type="text"
+    value={title}
+    readOnly={true}
+    placeholder="Input title"
+    style={{ textOverflow: 'ellipsis', background: 'red' }}
+  />
+</div>
+```
+
+Cela donne une nouvelle couleur de fond pour l'objet.
+
+![changement de fond de la tâche](/intro-to-storybook/chromatic-task-change.png)
+
+Ajoutez le fichier:
+
+```bash
+git add src\components\Task.js
+```
```

Comment:
@Draketheb4dass when you're able please adjust the snippet to "git add ." instead of  "git add src\components\Task.js", i updated the tutorial recently and introduced a misconception. I was working on a windows box and the snippet would work, but for other readers running linux or mac will not.